go.mod

```diff
@@ -5,4 +5,7 @@ go 1.13
 require (
 	github.com/facebook/fbthrift v0.31.1-0.20211129061412-801ed7f9f295
 	github.com/stretchr/testify v1.7.0
+	golang.org/x/net v0.5.0
 )
+
+replace github.com/facebook/fbthrift => github.com/vesoft-inc/fbthrift v0.0.0-20230201034936-5c5dd72a96c2
```

Comment:
We can just replace `github.com/facebook/fbthrift v0.31.1-0.20211129061412-801ed7f9f295` with `github.com/vesoft-inc/fbthrift v0.0.0-20230201034936-5c5dd72a96c2`